src/Microsoft.VisualStudio.Services.Agent/AgentServer.cs

```diff
@@ -46,6 +50,7 @@ public sealed class AgentServer : AgentService, IAgentServer
             }
 
             _taskAgentClient = _connection.GetClient<TaskAgentHttpClient>();
+            _releaseClient = _connection.GetClient<ReleaseHttpClient>();
```

Comment:
Missed to clean it up. I moved the apis to ReleaseServer but this was left out. Thanks for pointing out